resizer.go

```diff
@@ -443,7 +443,11 @@ func imageCalculations(o *Options, inWidth, inHeight int) float64 {
 	switch {
 	// Fixed width and height
 	case o.Width > 0 && o.Height > 0:
-		factor = math.Min(xfactor, yfactor)
+		if o.Crop {
```

Comment:
When `Crop` option is set, then we should resize less and then crop
When no `Crop` option is set, we resize more, to fit the resulting rectangle